developerportal/apps/events/models.py

```diff
@@ -330,7 +330,7 @@ class Event(Page):
     @property
     def is_upcoming(self):
         """Returns whether an event is in the future."""
-        return self.start_date > datetime.date.today()
+        return self.start_date >= datetime.date.today()
```

Comment:
This makes sense to stay as `start_date` (as opposed to the filtering above) but, being really pedantic, I think we should go back to `>`.

Reply:
I get what you're saying, but I think we can live with `>=` given that "today" is quite a bunch of waking hours across the globe, yeah?